bloomberg/xcdiff
# #119: Fix target attributes value mismatch

*Issue number of the reported bug or feature request: #119*

**Describe your changes**
SystemCapabilities attributes triggered value mismatch when the attributes where listed in different order.
Ordered both project and target attributes to ensure the comparison is not affected by the order of the attributes.

**Testing performed**
- Ensure all CI checks pass
- Inspect changes in snapshots files
- Copy the same project with SystemCapabilities attributes, re-order manually attributes in one of the projects and do the diff, verify xcdiff does not report any differences
